nixos/modules/services/web-servers/garage.nix

```diff
@@ -56,10 +59,12 @@ in
     };
 
     package = mkOption {
-      default = pkgs.garage;
-      defaultText = literalExpression "pkgs.garage";
+      # TODO: when 22.11 is released, put a stateVersion check on Garage 0.7.
+      default = if versionOlder stateVersion "23.05" then pkgs.garage_0_8_0
+                else pkgs.garage_0_7;
```

Comment:
22.11 is released, is this a reoccurring TODO?

Is the default set correctly? Might have a look at the `services.postgresql.package` definition.

Reply:
Need to verify ; the TODO is wrong indeed!

Reply:
Fixed!